handlers/registration.js

```diff
@@ -28,25 +31,55 @@ module.exports.create = async (event, ctx, callback) => {
   }
   const id = parseInt(data.id, 10);
   const eventID = data.eventID;
+  let eventName = "";
   let registrationStatus = data.registrationStatus;
 
   // Check if the event is full
-  if (registrationStatus === 'registered'){
+  if (registrationStatus === 'registered') {
 
     const eventParams = {
       Key: { id: eventID },
       TableName: 'biztechEvents' + process.env.ENVIRONMENT
     }
-    
+
     await docClient.get(eventParams).promise()
-    .then(async(event) => {
-      const counts = await helpers.getEventCounts(eventID)
+      .then(async (event) => {
+        const counts = await helpers.getEventCounts(eventID);
+        eventName = event.Item.ename;
+
+        if (counts.registeredCount >= event.Item.capac) {
+          registrationStatus = 'waitlist'
+        }
+      })
+  }
 
-      if (counts.registeredCount >= event.Item.capac){
-        registrationStatus = 'waitlist'
+  //confirmation email 
+  const userParams = {
+    Key: { id: id },
+    TableName: 'biztechUsers' + process.env.ENVIRONMENT
+  }
+
+  await docClient.get(userParams).promise()
+    .then(async (user) => {
+      console.log(user);
+      const userEmail = await user.Item.email;
+      const userName = await user.Item.fname;
+      if (eventName != undefined) {
```

Comment:
You could chain the `biztechUsers` table read in a `.then` after the `biztechEvents` to avoid this eventName checking and such. if you think that's cleaner. I don't know

Reply:
Ok I used .then instead. I think it looks a lot cleaner and makes more sense.